frontend/server/src/main/java/org/pytorch/serve/metrics/MetricBuilder.java

```diff
@@ -0,0 +1,51 @@
+package org.pytorch.serve.metrics;
+
+import java.util.ArrayList;
+import org.pytorch.serve.metrics.format.prometheous.PrometheusCounter;
+import org.pytorch.serve.metrics.format.prometheous.PrometheusGauge;
+import org.pytorch.serve.metrics.format.prometheous.PrometheusHistogram;
+
+public final class MetricBuilder {
```

Comment:
Please make this as singleton

Reply:
There is no use case to instantiate this class so I've skipped the singleton implementation.

Reply:
Since this is a utility class, we won't instantiate it, so no constructor is required. If no constructor is provided, the default `public` constructor will be added which leads to the following checkstyle error: `Utility classes should not have a public or default constructor. [HideUtilityClassConstructor]`. Therefore, the dummy private constructor is added. It is not intended to be called. Similar approach is used for example in [NettyUtils.java](https://github.com/pytorch/serve/blob/master/frontend/server/src/main/java/org/pytorch/serve/util/NettyUtils.java#L69)